UI updated.

u7i opened the pull request:
Optimized interface size, now it's much more intuitive.

New UI - https://justpaste.it/6wnml
Old UI - https://justpaste.it/3lrts

umnicode committed:
UI updated.
Optimized interface size, now it's much more intuitive.

New UI - https://justpaste.it/6wnml
Old UI - https://justpaste.it/3lrts

umnicode committed:
- Fixed blinking on video changing.
- Fixed bug in playback-rate slider

u7i commented:
In the second commit, I have fixed the blinking that appeared when the video reached its end. Also I have fixed playback-rate slider

halverneus commented:
This is looking really good. I'll try it out this weekend. Feel free to continue adding to the merge request, in the mean time.

u7i commented:
> This is looking really good. I'll try it out this weekend. Feel free to continue adding to the merge request, in the mean time.

Thank you

umnicode committed:
- Playlists support enabled
- Read-Me updated
- TODO list added
- Wallpaper drawer fully rewrited

For now, this is the last major commit in the merge request.
I have fixed all major problems that I could find

If you encount any problems: open issue or write to my emal:
umnicodeusa@gmail.com

umnicode committed:
- Fixing errors after git-merge

umnicode committed:
Grammar typo fixing
